docker_test.go

```diff
@@ -1087,6 +1108,56 @@ func TestContainerCreationWithVolumeAndFileWritingToIt(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestContainerCreationWithVolumeCleaning(t *testing.T) {
+	absPath, err := filepath.Abs(filepath.Join(".", "testdata", "hello.sh"))
+	require.NoError(t, err)
```

Comment:
Just so you're aware (no need to change) we've been using embed for this type recently to its easier to reuse.
```go
//go:embed testdata/hello.sh
var helloSh []byte
```
Then use:
```go
ContainerFile{
    Reader: bytes.NewReader(helloSh),
    ContainerFilePath: "/hello.sh",
    FileMode: 0o755,
}
```